README.md

```diff
@@ -1,9 +1,21 @@
-# Apostrophe 3 demo and test project
+# Apostrophe 3 essentials starter kit
 
-## Get started
+## Getting started
 
-If you created your project using the CLI tool, all dependencies will already have been installed, so you can skip this step. Otherwise, install 
-the dependencies using: `npm install`
+This Starter Kit serves as a template for initiating new projects and is intended for use in two main ways:
+
+1. **Using Our CLI Tool**: Run our CLI tool to clone this template locally, install its dependencies, and set up an initial admin user.
+  
+2. **Manual Setup**: Manually clone this repository and install its dependencies using `npm install`. Add an initial admin user with `node app @apostrophecms/user:add admin admin`.
```

Comment:
You can fork and rename in one step, so I'd link to github docs on how to "fork and rename" the repository, rather than referring to cloning it but not saying anything about how. That could lead to something like manually creating your own copy without `git clone` or something.

Reply:
Or... just say "manually `git clone` the repository" and don't go overboard. I just want people to get roughly the same amount of guidance you're giving them on the steps that follow this step.

Reply:
Changed.